infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesWorkspaceServiceAccount.java

```diff
@@ -13,12 +13,12 @@
 
 import static com.google.common.base.Strings.isNullOrEmpty;
 
-import io.fabric8.kubernetes.api.model.rbac.KubernetesPolicyRuleBuilder;
-import io.fabric8.kubernetes.api.model.rbac.KubernetesRole;
-import io.fabric8.kubernetes.api.model.rbac.KubernetesRoleBinding;
-import io.fabric8.kubernetes.api.model.rbac.KubernetesRoleBindingBuilder;
-import io.fabric8.kubernetes.api.model.rbac.KubernetesRoleBuilder;
-import io.fabric8.kubernetes.api.model.rbac.KubernetesSubjectBuilder;
+import io.fabric8.kubernetes.api.model.rbac.PolicyRuleBuilder;
```

Comment:
It would be interesting to check if with these changes we're able to get rid of https://github.com/eclipse/che/blob/master/infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftWorkspaceServiceAccount.java

Reply:
maybe in separate task?

Reply:
Sure, just a proposal and I'm fully OK with any option, even - maybe never ;)